File: Beşgün Turizm Web/webVTYSvize/Data/Otel.cs
using Microsoft.AspNetCore.Mvc;
using static System.Net.Mime.MediaTypeNames;
using System.Drawing;
using System.ComponentModel.DataAnnotations;

namespace webVTYSvize.Data
{
    public class Otel
    {

        public Otel(int idOtel, string adres,int HizmetBilgisi_idHizmetBilgisi)
        {
            this.idOtel = idOtel;
            this.adres = adres;
            this.HizmetBilgisi_idHizmetBilgisi = HizmetBilgisi_idHizmetBilgisi;

        }
        [Key]
        public int idOtel  { get; set; }
        public string adres { get; set; }
        public int HizmetBilgisi_idHizmetBilgisi { get; set; }
        public HizmetBilgisi? hizmetBilgisi { get; set; }
        public double puan { get; set; }
        public string tema { get; set; }


    }
}

File: Beşgün Turizm Mobil/app/src/main/java/com/muhammetcakir/turizmacentasi/Models/Otel.kt
package com.muhammetcakir.turizmacentasi.Models

data class Otel(
    val idOtel: Int,
    val adres: String,
    val HizmetBilgisi_idHizmetBilgisi: String,
    val puan:Double,
    val tema:String
)

data class ApiResponseOtel(
    val error: Boolean,
    val otel: List<Otel>
)

data class ApiResponseYurtD(
    val error: Boolean,
    val yurtdisitur: List<YurtdisiTur>
)
data class ApiResponseTekneT(
    val error: Boolean,
    val teknetur: List<TekneTur>
)

data class ApiResponseHizmetBilgisi(
    val error: Boolean,
    val hizmetbilgisi: List<HizmetBilgisi>
)
data class ApiResponseTurProgrami(
    val error: Boolean,
    val turprogrami: List<TurProgrami>
)
data class ApiResponseHizmet(
    val error: Boolean,
    val hizmet: List<Hizmet>
)
data class ApiResponseMusteri(
    val error: Boolean,
    val musteri: List<Musteri>
)
data class ApiResponseRezervasyon(
    val error: Boolean,
    val rezervasyon: List<Rezervasyon>
)
data class ApiResponseOtelView(
    val error: Boolean,
    val otel: List<OtelView>
)
data class ApiResponseYurtDisiTurView(
    val error: Boolean,
    val yurtdisitur: List<YurtDisiTurView>
)
data class ApiResponseTekneTurView(
    val error: Boolean,
    val teknetur: List<TekneTurView>
)